src/ingestion_data.py

```diff
@@ -16,6 +25,16 @@ def __init__(self, json_data):
         self._payload = json_data
         self._sec = None
         self._update_sec = None
+        self._cves = self._get_cves()
+        if self._payload['body']:
+            self._payload['body'] = self._payload['body'][0:MAX_STRING_LENGTH]
+
+    def _get_cves(self):
+        """Get the CVEs mentioned from title or body."""
+        txt = self._payload['title']
+        if self._payload['body']:
+            txt = txt + self._payload['body']
+        return get_cves_from_text(txt)
```

Comment:
Usually the large text concat op would be expensive in terms of memory & CPU. In this you can do isolated text extraction.
```suggestion
        cves = get_cves_from_text (self._payload['title'])
        if self._payload['body']:
            cves.append(get_cves_from_text(self._payload['body']))
        return set(cves)
```

Reply:
done